src/dotnet/Fable.Compiler/Transforms/Replacements.fs

```diff
@@ -601,10 +597,6 @@ let rec equals (com: ICompiler) r equal (left: Expr) (right: Expr) =
         Helper.CoreCall("Reflection", "equals", Boolean, [left; right], ?loc=r) |> is equal
     | Tuple _ ->
         Helper.CoreCall("Util", "equalArrays", Boolean, [left; right], ?loc=r) |> is equal
-    | DeclaredType(ent,_) when hasBaseImplementingBasicMethods ent ->
-        Helper.InstanceCall(left, "Equals", Boolean, [right]) |> is equal
```

Comment:
Maybe we can leave this for optimization? It'll be faster to call `.Equals` directly.

Reply:
Right, same for classes.